src/controllers/user.controller.js

```diff
@@ -0,0 +1,95 @@
+const userService = require('../services/user.service');
+
+const get = async (req, res) => {
+  const users = await userService.getAll();
+
+  if (!users) {
+    res.send(users);
+    res.sendStatus(404);
```

Comment:
There is a logical error here. You are sending a response with 'users' and then immediately sending a 404 status. The 'res.sendStatus(404)' line will not be executed because 'res.send(users)' already sends the response. Consider removing 'res.send(users)' if you intend to send a 404 status when no users are found.